android/expoview/src/main/java/host/exp/exponent/notifications/helpers/ExpoCronParser.java

```diff
@@ -0,0 +1,62 @@
+package host.exp.exponent.notifications.helpers;
+
+import com.cronutils.builder.CronBuilder;
+import com.cronutils.model.Cron;
+
+import java.util.HashMap;
+
+import static com.cronutils.model.field.expression.FieldExpressionFactory.always;
+import static com.cronutils.model.field.expression.FieldExpressionFactory.on;
+import static com.cronutils.model.field.expression.FieldExpressionFactory.questionMark;
+
+public class ExpoCronParser {
+  public static Cron createCronInstance(HashMap<String, Object> options) {
+    CronBuilder cronBuilder = CronBuilder.cron(ExpoCronDefinitionBuilder.getCronDefinition());
+
+    if (options.containsKey("year")) {
```

Comment:
In general I would say it's safer to do `if (options.get("year") instanceof Number)` cause we're checking for null and type cast at the same time